examples/platforms/nrf52840/platform-config.h

```diff
@@ -48,6 +48,16 @@
  * @section UART Driver Configuration.
  ******************************************************************************/
 
+/**
```

Comment:
All looks good, just one more nit.

Probably i wasn't so clear on last my comment. Currently this define is under `@section UART Driver Configuration.`
Could you please create a new section e.g: `@section Platform FEM Configuration` and place it close to our radio driver configuration.